designer/client/src/actions/nk/process.ts

```diff
@@ -30,6 +30,18 @@ export function fetchProcessToDisplay(processName: ProcessName, versionId?: Proc
     };
 }
 
+export function fetchValidatedProcess(processName: ProcessName, versionId?: ProcessVersionId): ThunkAction<Promise<Scenario>> {
```

Comment:
Do we need it? If not, could you also remove [import](https://github.com/TouK/nussknacker/pull/7515/files#diff-4c59f0eb5ae009edd55ab4cbba148ea5f811f92ab15ee6e10cdcd9877fdffd92R2) of it?